sqld/src/lib.rs

```diff
@@ -305,6 +305,11 @@ fn configure_rpc(config: &Config) -> anyhow::Result<(Channel, tonic::transport::
     Ok((channel, uri))
 }
 
+pub enum ResetOp {
+    Reset(Bytes),
+    Destroy(Bytes),
```

Comment:
Why is it called `ResetOp`? Is destroying the database considered special case of reset? Are there any future operations that could eventually use the same mechanism this enum is used in?

Reply:
There are two cases: when a replica is incompatible it is reset to replicate again, when a namespace doesn't exist, it is destroyed completely. This is the adaptation of old code, and it could probably use a refactor